benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -13,13 +13,14 @@
 
 r"""Microbenchmarks for the quantized activations."""
 
-qrelu_configs = op_bench.cross_product_configs(
+qactivation_configs = op_bench.cross_product_configs(
     dims=(
```

Comment:
Do you need to do cross product of all the Ranks? In the current way, only one dims will be tested.

Reply:
Oh I wanted tot check the combinations of every dimension with every other parameter. The `cross_product_configs` is not the right way to go?

Reply:
I mean, I need to check one dim at a time. For example one example to benchmarrk would be

```
dims = (1, 3, 4, 5)
permute=False
inplace=True
```

and another

```
dims = (4, 1, 1, 1)
permute=False
inplace=True
```

Reply:
In this example, the cross-product function will just generate {dims} * {permute_dims} * {inplace} which is 1 * 2 * 2 = 4 different combinations. This doesn't seem to be what you want.  

If you want to generate cross product of dims, the dims needs to be split. For example, 
op_bench.cross_product_configs(
rank1 = (1,), (1, 1), (1, 1, 1),
rank2 = (2, 1), (1, 2), 
rank3 = (3, 4, 5)
...
}

This will generate cross product of {rank1} * {rank2} * {rank3} = 3 * 2 * 1 which is total of 6 different input shapes.

Reply:
You are right. I misunderstood what you were trying to do here.